src/FlubuCore/Targeting/TargetTree.cs

```diff
@@ -288,33 +290,48 @@ public virtual void LogBuildSummary(IFlubuSession session)
                 }
             }
 
-            int maxTargetNameLength;
-
             if (session.Args.DryRun)
             {
                 session.LogInfo("DRY RUN PERFORMED");
             }
             else if (session.UnknownTarget.HasValue && !session.UnknownTarget.Value)
             {
-                var targets = GetTargetsInExecutionOrder(session);
```

Comment:
Code block from here is refactored to ode block from here is refactored to method `LogBuildSummaryTable`